Changelog

```diff
@@ -1,3 +1,42 @@
+## Version 1.3.0
+-------------------------------------------------------------------------------
+
+License
+  - Change the license from MIT to BSD-3-Clause
+
+New features
+  - Support to set separate log directory to store server logs (#155)
+  - Implement PSETEX command (#156)
+  - Implement MSETNX command (#165)
+
+Improvements
+  - Improve the precision of the INCRBYFLOAT/HINCRBYFLOAT command to double (#169)
+  - Support +INF/-INF for the score of zset type members (#172)
+  - The UNSUBSCRIBE and PUNSUBSCRIBE commands support multiple channels or patterns (#174)
+
+Behavior changes
+  - The UNSUBSCRIBE and PUNSUBSCRIBE commands have the same reply as Redis (#174)
+  - The same message format as Redis for pattern subscribers when publish message (#175)
+    Message foromat is from 'message channel message' to 'pmessage pattern channel message'
+
+Bug fixes
+  - Fix skipping some SST files when pick compaction files (#154)
+  - Handle negative expire time in set command to avoid deleting keys (#156)
+  - List type command doesn't rewrite other types data (#157)
```

Comment:
List type command doesn't rewrite other types data  => List type command doesn't rewrite other types of data?